org.eclipse.ice.commands/src/main/java/org/eclipse/ice/commands/LocalMoveFileCommand.java

```diff
@@ -68,45 +68,62 @@ public CommandStatus execute() {
 	@Override
 	protected CommandStatus run() {
 
-		// Check to see if the file paths are the same and if the filename has the same extension
-		// for determining whether or not it should be a move or change file name
+		// Check to see if the file paths are the same and if the filename has the same
+		// extension for determining whether or not it should be a move or change file
+		// name
 		boolean sameFileExtension = checkFileExtension();
 
 		// Split the destination by path delimiter to get the desired filename
 		String[] destinationDirs = destination.toString().split("/");
-		
+
 		// If the directory paths are the same and the files have the same file
 		// extension
 		// type, then we are just changing a file name
 		if (sameFileExtension) {
 			try {
+				status = CommandStatus.RUNNING;
 				// destinationDirs will have the desired file name in the length-1 entry
 				Files.move(source, source.resolveSibling(destinationDirs[destinationDirs.length - 1]),
 						REPLACE_EXISTING);
 			} catch (IOException e) {
 				e.printStackTrace();
```

Comment:
Remove or log.

Reply:
See above comment, I went through and made sure every exception was logged.